server/controllers/find/coursesController.test.ts

```diff
@@ -42,4 +46,38 @@ describe('CoursesController', () => {
       expect(courseService.getCourses).toHaveBeenCalledWith(token)
     })
   })
+
+  describe('show', () => {
+    it('show render the course show template', async () => {
+      const course = courseFactory.build()
+      courseService.getCourse.mockResolvedValue(course)
+
+      const courseOfferings = courseOfferingFactory.buildList(10)
```

Comment:
Make this 3 to reduce the looping given we aren't testing anything related to the number